pkg/apis/ceph.rook.io/v1/scc.go

```diff
@@ -26,15 +27,15 @@ import (
 
 // NewSecurityContextConstraints returns a new SecurityContextConstraints for Rook-Ceph to run on
 // OpenShift.
-func NewSecurityContextConstraints(name, namespace string) *secv1.SecurityContextConstraints {
+// namespaces: can be single namespace or comma seprated list of namespaces
+func NewSecurityContextConstraints(name, namespaces string) *secv1.SecurityContextConstraints {
```

Comment:
you are still inputting one only namespace

Reply:
ah okay, you are passing it as a string and splitting it, why not use Variadic Functions?

Reply:
Variadic Functions makes more sense. done